manifests/site.pp

```diff
@@ -21,6 +21,9 @@
 if $ec2_tag_env {
   $env = $ec2_tag_env
 }
+if $ec2_tag_name {
+  $host_name = $ec2_tag_name
```

Comment:
Where is `$host_name` used?

Reply:
To get the Name tag from aws and set it as its hostname

Reply:
But I don't see that variable used anywhere in our code.

Reply:
(And hostname is one word, for what it's worth.)

Reply:
`set_hostname() {
  instance_id=$(curl 169.254.169.254/latest/meta-data/instance-id)
  host_name=${hostname}-$${instance_id}
  domain=${environment}.graduway.com
  fqdn=$${host_name}.$${domain}
  hostnamectl set-hostname "$${fqdn}"
  echo 127.0.0.1 $${host_name} $${fqdn} >> /etc/hosts
}`
it uses it during boot proccess using terraform, so you configure the name tag and it causes the linux host name to be aligned with that